src/api/Feed.js

```diff
@@ -645,10 +647,15 @@ class Feed extends Base {
 
         try {
             const taskLink = await this.createTaskLink(file, task);
-            const taskAssignments: Array<TaskCollabAssignee> = await Promise.all(
-                assignees.map((assignee: SelectorItem) => {
-                    return this.createTaskCollaborator(file, task, assignee);
-                }),
+            const taskAssignments: Array<TaskCollabAssignee> = flatten<TaskCollabAssignee, TaskCollabAssignee>(
+                await Promise.all(
```

Comment:
Rather than doing these in parallel, we should do something like Promise.all filter for individual collaborators first and then do Promise.all filter down to groups. That way the group endpoints de-dupes